src/push.rs

```diff
@@ -80,6 +80,10 @@ fn push(job: &str,
         method: &str)
         -> Result<()> {
 
+    // Suppress clippy warning needless_pass_by_value.
+    let grouping = grouping;
+    let mfs = mfs;
```

Comment:
Sure, but taking a reference will change APIs. To fix it, I am going to send a PR separately.